src/Plugin/Field/FieldFormatter/GoogleFormFormatter.php

```diff
@@ -22,6 +22,22 @@
  */
 class GoogleFormFormatter extends FormatterBase {
 
+  /**
+   * The name of the iframe height field.
+   *
+   * @var string
+   */
+  protected $iframeHeightField;
+
+  /**
+   * {@inheritDoc}
+   */
+  public function __construct($plugin_id, $plugin_definition, FieldDefinitionInterface $field_definition, array $settings, $label, $view_mode, array $third_party_settings) {
+    parent::__construct($plugin_id, $plugin_definition, $field_definition, $settings, $label, $view_mode, $third_party_settings);
+    $media_type = MediaType::load($field_definition->getTargetBundle());
+    $this->iframeHeightField = $media_type->getSource()->getConfiguration()['height_field_name'];
```

Comment:
What you could do here is add a method on the source plugin `getHeightField`. it would make it more oop.

Reply:
I actually did that at first, and then I removed the `getHeightField` function, since I never ended up using it.

Reply:
Created the additional function, and set it up in place of this configuration call.